core/src/main/java/com/linecorp/armeria/server/AbstractPathMapping.java

```diff
@@ -84,7 +84,7 @@ public String loggerName() {
         return "__UNKNOWN__";
     }
 
-    static String loggerName(String pathish) {
+    protected static String loggerName(String pathish) {
```

Comment:
Then could you add a Javadoc? We may eventually remove this though, given that `ServiceRequestContext.logger()` has been deprecated.